model/DataStore/MetaDataDeliverySyncTask.php

```diff
@@ -159,10 +158,9 @@ private function getTest(core_kernel_classes_Resource $deliveryResource): core_k
     private function getTestUri(core_kernel_classes_Resource $deliveryResource): ?string
     {
         $testProperty = $this->getProperty(DeliveryAssemblyService::PROPERTY_ORIGIN);
+        $test = $deliveryResource->getOnePropertyValue($testProperty);
 
-        return ($deliveryResource->getOnePropertyValue($testProperty)) ?
-            $deliveryResource->getOnePropertyValue($testProperty)->getUri() :
-            null;
+        return ($test) ? $test->getUri() : null;
```

Comment:
No need for parenthesis
```suggestion
        return $test ? $test->getUri() : null;
```